packages/fluentui/react-bindings/src/utils/mergeVariablesOverrides.ts

```diff
@@ -0,0 +1,13 @@
+import { ComponentVariablesInput, mergeComponentVariables } from '@fluentui/styles';
+
+/**
+ mergeComponentVariables() is always creating a function even if the arguments are undefined we have this temporary
+ fix in place to avoid creating empty function because it is breaking caching we should either fix
+ mergeComponentVariables(), or handle this in a more generic way.
+ */
+export default function mergeVariablesOverrides(
```

Comment:
should we add this under `@fluentui/styles` with the other merge functions?

Reply:
I don't have there any preference, however I placed it here as it's more attached to component's specifics